app/views/devise/sessions/new.html.erb

```diff
@@ -15,7 +15,7 @@
     <div class="form-check">
       <%= f.label :remember_me, class: 'form-check-label' do %>
         <%= f.check_box :remember_me, class: 'form-check-input' %>
-        <%= t('.remember_me', default: 'Remember me') %>
+        <%= t('activerecord.attributes.user.remember_me', default: 'Remember me') %>
```

Comment:
This string should not be model specific, it will not work for custom names like admin.